training/deepspeech_training/util/sample_collections.py

```diff
@@ -207,7 +214,7 @@ def __init__(self, sdb_filename, buffering=BUFFER_SIZE, id_prefix=None, labeled=
         """
         self.sdb_filename = sdb_filename
         self.id_prefix = sdb_filename if id_prefix is None else id_prefix
-        self.sdb_file = open(sdb_filename, 'rb', buffering=buffering)
+        self.sdb_file = open(sdb_filename, 'rb', buffering=REVERSE_BUFFER_SIZE if reverse else buffering)
```

Comment:
Parameter is ignored.]
```suggestion
        self.sdb_file = open(sdb_filename, 'rb', buffering=reverse_buffering if reverse else buffering)
```

Reply:
Nevermind, I see you removed the parameter instead.